Formula/mongodb-community.rb

```diff
@@ -5,11 +5,11 @@ class MongodbCommunity < Formula
   # frozen_string_literal: true
 
   if Hardware::CPU.intel?
```

Comment:
This file is a copy of `Formula/mongodb-community@6.0.rb` with the diff being the only thing that changed